docs/product-user-guide/src/main/asciidoc/drl-rules-other-con.adoc

```diff
@@ -0,0 +1,13 @@
+[id='drl-rules-other-con']
+= Other methods for creating DRL files
+
+As an alternative to creating and managing DRL files within the {CENTRAL} interface, you can create DRL files externally using Red Hat Developer Studio, Java objects, or Maven archetypes. These DRL files can then be integrated with your {PRODUCT} decision service, so long as the external DRL files contain at minimum the required `package` specification, `import` lists, and `rule` definitions. Any other DRL components, such as global variables and functions, are optional. All data objects related to a DRL rule must be included with your external DRL project or deployment.
+
+*<@Michael (10 of 13): I pulled the content in this chapter from all over our Dev Guide, cleaned them up, and added to them. A few general questions for you:>*
```

Comment:
@manstis, getting started with your feedback now, but wanted to just mention here, before I dive in elsewhere, that I was/am on the fence about these sections and appreciate your thoughts. The only reason I included them is that I discussed it with our team content strategist and the thinking was that because this doc is about using DRL rules to design a decision service, it made sense to at least mention methods other than Central for creating DRL files. I'll definitely mention at the end of each section of maybe the beginning of the chapter the things you've enlightened me about for using standalone projects like this. 

That said, do you feel strongly that these really shouldn't even be mentioned at all in this doc? If so, I can re-bring it up with out content strategist.

Reply:
Hi @sterobin in the old world, sections related to features, for example the "DRL Editor". However in the brave new world, as far as I gather, sections cut across features covering use cases. Given that, then I guess having the standalone use examples, sort of fit here as they cover DRL. However why does the Guided Decision Table editor not cover XLS Tables too - and to draw a comparison - using XLS Decision Tables from Java code and not BC? LIkewise with Templates etc.. IIRC the new sections for those did not cover how to use, for example, a Template from Java code in standalone use of Drools? So the inclusion here is perhaps inconsistent?

I'm not a technical writer however, so if the content strategists think here is the best place for use of DRL standalone Java code I won't argue! Personally I'd have thought their inclusion in an integration guide makes more sense - where we'd document using either KIE\Decision Server or stanalone/embedded.